### ISSUE TITLE
CI failure in JIT.Regression Runtime_40444.cmd

### ISSUE THREAD

**Issue by tarekgh** [dotnet/runtime#44831]
https://dev.azure.com/dnceng/public/_build/results?buildId=889393&view=logs&j=cc37abc9-5ced-5ca3-7db5-cfbd415ba586&t=6ce0ccaa-325b-5477-c8ab-ffa0b32a6fdc

```
C:\h\w\A75608B4\w\A9D3095D\e>dotnet C:\h\w\A75608B4\p\xunit.console.dll JIT\Regression\JIT.Regression.XUnitWrapper.dll -parallel collections -nocolor -noshadow -xml testResults.xml -trait TestGroup=JIT.Regression 
Microsoft.DotNet.XUnitConsoleRunner v2.5.0 (32-bit .NET 5.0.0)
  Discovering: JIT.Regression.XUnitWrapper
  Discovered:  JIT.Regression.XUnitWrapper
  Starting:    JIT.Regression.XUnitWrapper
    JIT\Regression\JitBlue\Runtime_40444\Runtime_40444\Runtime_40444.cmd [FAIL]
      
      Return code:      1
      Raw output file:      C:\h\w\A75608B4\w\A9D3095D\e\JIT\Regression\Reports\JIT.Regression\JitBlue\Runtime_40444\Runtime_40444\Runtime_40444.output.txt
      Raw output:
      BEGIN EXECUTION
       "C:\h\w\A75608B4\p\corerun.exe" Runtime_40444.dll 
      0: result = 1
      0: result = 2
      0: result = 3
      0: result = 4
      0: result = 5
      0: result = 6
      0: result = 7
      0: result = 8
      0: result = 9
      0: result = 10
      0: result = 11
      0: result = 12
      0: result = 13
      0: result = 14
      0: result = 15
      0: result = 16
      0: result = 17
      0: result = 18
      0: result = 19
      0: result = 20
      0: result = 21
      0: result = 22
      0: result = 23
      0: result = 24
      0: result = 25
      0: result = 26
      0: result = 27
      0: result = 28
      0: result = 29
      0: result = 30
      0: result = 31
      0: result = 32
      0: result = 33
      0: result = 34
      0: result = 35
      0: result = 36
      0: result = 37
      0: result = 38
      0: result = 39
      0: result = 40
      0: result = 41
      0: result = 42
      0: result = 43
      0: result = 44
      0: result = 45
      0: result = 46
      0: result = 47
      0: result = 48
      0: result = 49
      0: result = 50
      0: result = 51
      0: result = 52
      0: result = 53
      0: result = 54
      0: result = 55
      0: result = 56
      0: result = 57
      0: result = 58
      8732: result = 59
      0: result = 60
      0: result = 61
      0: result = 62
      0: result = 63
      0: result = 64
      0: result = 65
      0: result = 66
      0: result = 67
      0: result = 68
      0: result = 69
      0: result = 70
      0: result = 71
      0: result = 72
      0: result = 73
      0: result = 74
      0: result = 75
      9546: result = 76
      0: result = 77
      8440: result = 78
      0: result = 79
      8043: result = 80
      0: result = 81
      8156: result = 82
      0: result = 83
      14: result = 84
      0: result = 85
      0: result = 86
      0: result = 87
      0: result = 88
      FAILED
      Expected: 100
      Actual: -1
      END EXECUTION - FAILED
      FAILED
      Test Harness Exitcode is : 1
      To run the test:
      > set CORE_ROOT=C:\h\w\A75608B4\p
      > C:\h\w\A75608B4\w\A9D3095D\e\JIT\Regression\JitBlue\Runtime_40444\Runtime_40444\Runtime_40444.cmd
      Expected: True
      Actual:   False
      Stack Trace:
        F:\workspace\_work\1\s\artifacts\tests\coreclr\Windows_NT.x86.Checked\TestWrappers\JIT.Regression\JIT.Regression.XUnitWrapper.cs(56691,0): at JIT_Regression._JitBlue_Runtime_40444_Runtime_40444_Runtime_40444_._JitBlue_Runtime_40444_Runtime_40444_Runtime_40444_cmd()
      Output:
        
        Return code:      1
        Raw output file:      C:\h\w\A75608B4\w\A9D3095D\e\JIT\Regression\Reports\JIT.Regression\JitBlue\Runtime_40444\Runtime_40444\Runtime_40444.output.txt
        Raw output:
        BEGIN EXECUTION
         "C:\h\w\A75608B4\p\corerun.exe" Runtime_40444.dll 
        0: result = 1
        0: result = 2
        0: result = 3
        0: result = 4
        0: result = 5
        0: result = 6
        0: result = 7
        0: result = 8
        0: result = 9
        0: result = 10
        0: result = 11
        0: result = 12
        0: result = 13
        0: result = 14
        0: result = 15
        0: result = 16
        0: result = 17
        0: result = 18
        0: result = 19
        0: result = 20
        0: result = 21
        0: result = 22
        0: result = 23
        0: result = 24
        0: result = 25
        0: result = 26
        0: result = 27
        0: result = 28
        0: result = 29
        0: result = 30
        0: result = 31
        0: result = 32
        0: result = 33
        0: result = 34
        0: result = 35
        0: result = 36
        0: result = 37
        0: result = 38
        0: result = 39
        0: result = 40
        0: result = 41
        0: result = 42
        0: result = 43
        0: result = 44
        0: result = 45
        0: result = 46
        0: result = 47
        0: result = 48
        0: result = 49
        0: result = 50
        0: result = 51
        0: result = 52
        0: result = 53
        0: result = 54
        0: result = 55
        0: result = 56
        0: result = 57
        0: result = 58
        8732: result = 59
        0: result = 60
        0: result = 61
        0: result = 62
        0: result = 63
        0: result = 64
        0: result = 65
        0: result = 66
        0: result = 67
        0: result = 68
        0: result = 69
        0: result = 70
        0: result = 71
        0: result = 72
        0: result = 73
        0: result = 74
        0: result = 75
        9546: result = 76
        0: result = 77
        8440: result = 78
        0: result = 79
        8043: result = 80
        0: result = 81
        8156: result = 82
        0: result = 83
        14: result = 84
        0: result = 85
        0: result = 86
        0: result = 87
        0: result = 88
        FAILED
        Expected: 100
        Actual: -1
        END EXECUTION - FAILED
        FAILED
        Test Harness Exitcode is : 1
        To run the test:
        > set CORE_ROOT=C:\h\w\A75608B4\p
        > C:\h\w\A75608B4\w\A9D3095D\e\JIT\Regression\JitBlue\Runtime_40444\Runtime_40444\Runtime_40444.cmd
```

**Comment by echesakovMSFT** [comment]
cc @briansull

**Comment by briansull** [comment]
The version of this test was updated (by #40885), this job is running the old version of this test.

**Comment by tarekgh** [comment]
@briansull this failure on 5.0 branch. Could you update the test there? otherwise it will fail in most of the jobs there.

**Comment by briansull** [comment]
@tarekgh Do you have a pointer to instructions on how to port to release\5.0
I tried a few different things and I didn't get it to work for me.

Basically we just want to do "git cherry-pick  ead6ddf4a70f9505" and resolve the conflict in issues.targets

**Comment by tarekgh** [comment]
@briansull if this test changed in isolated PR in master, then you may do something like in the comment https://github.com/dotnet/runtime/pull/44732#issuecomment-729928234. You can also do it manually by pulling locally release\5.0 branch and then cherry pick the change. either way, please inform @jeffschwMSFT about this port as I am not sure what is the policy for porting test fixes to release branches.

**Comment by JulieLeeMSFT** [comment]
Will work with @briansull offline to backport this infrastructure related changes.

**Comment by danmosemsft** [comment]
Hit again. https://dev.azure.com/dnceng/public/_build/results?buildId=894293&view=ms.vss-test-web.build-test-results-tab&runId=28587676&paneView=debug&resultId=101521

**Comment by danmosemsft** [comment]
Another one. https://dev.azure.com/dnceng/public/_build/results?buildId=898236&view=ms.vss-test-web.build-test-results-tab&runId=28699932&resultId=101607&paneView=debug

**Comment by danmosemsft** [comment]
A 3rd failure today.
https://dev.azure.com/dnceng/public/_build/results?buildId=898236&view=ms.vss-test-web.build-test-results-tab&runId=28699932&resultId=101607&paneView=debug

**Comment by briansull** [comment]
The PR that contains the fix for this issue is #40951

**Comment by briansull** [comment]
Unfortunately the automated backport to 5.0 fails for this PR:

>backporting to release/5.0 failed, the patch most likely resulted in conflicts:

**Comment by briansull** [comment]
@AndyAyersMS Could you help me check this change into release\5.0

I have staged a fix here: 

https://github.com/briansull/runtime/tree/fix-44831-5.0

**Comment by AndyAyersMS** [comment]
Helped Brian get unblocked, PR for 5.0 is #45431

**Comment by briansull** [comment]
Manual run of jit-stress to confirm that it is green:

https://dev.azure.com/dnceng/public/_build/results?buildId=906223&view=results

**Comment by briansull** [comment]
Fixed with #45341
